client/web/compose/src/components/PageBlocks/GeometryBase.vue

```diff
@@ -149,7 +149,7 @@ export default {
         zoomMax,
       }
 
-      Promise.all(this.options.feeds.map((feed, idx) => {
+      Promise.all(this.options.feeds.filter(({ options }) => options.moduleID !== NoID).map((feed, idx) => {
```

Comment:
I doubt this is the only thing needed for it to function no? You also need other configurations so, I would suggest you add a isValid method to the feeds and do it like this.options.feed.filter(f => f.isValid())